v2/context/backoff.go

```diff
@@ -0,0 +1,24 @@
+package context
+
+import "time"
+
+type BackoffStrategy string
+
+const (
+	BackoffStrategyLinear      = "linear"
+	BackoffStrategyExponential = "exponential"
+)
+
+// Backoff holds parameters applied to retries
+type Backoff struct {
```

Comment:
Perhaps this should be called `Retries` or `RetryParams`?

Reply:
yes make sense. Done.